### DISCUSSION TITLE
Results on COCO test set? Comparison to YOLOv8/10/11/12?

### DISCUSSION THREAD

**Discussion by Krystex** [Lillthorin/YoloLite-Official-Repo#9] [Ideas]
Hi there! 
This project looks really promising, I will take a closer look if I have the time!

I took a quick look a the benchmark results and they look quite good for such a small model.
However I couldn't find any information about performance on the COCO test set, did you try to train on MSCOCO yet? That way the model could be compared to other models more easily. Also pre-trained weights will be useful for finetuning on smaller datasets.

Also, do you know the mAP metrics of newer YOLO models like YOLOv8, YOLOv10, YOLOv11, YOLOv12, YOLOE? It would be quite interesting to see those.

Anyway, great work and have a nice day:)

**Comment by Lillthorin** [comment]
Hi! 

Thanks! I haven´t done any training on COCO, I know this would be the ideal benchmark but these small models will probably not do to well, RF100 offers a better overview of how the models perform on different tasks. Also the training will probably take days or weeks given my current setup. The backbones gets loaded with pretrained weights via timm, therefore you should be able to get quick results even on small datasets. All training for benchmark results was done with 100 epochs. 

The roboflow team has released numbers for Yolov8 on RF100, they can be found here: https://github.com/roboflow/roboflow-100-benchmark
I cannot find any table with data per each dataset as per yolov5 and yolov7, if they release it I will update my table. 

The edge models that are the main focus of this repo won´t compare to the bigger models but offers fast CPU performance while trying to minimize the accuary loss.
